container/src/container/array_dict.py

```diff
@@ -9,196 +9,194 @@ def isiterable(obj):
         return False
 
 class array_dict:
```

Comment:
Why not use Struct arrays or Pandas DataFrame?
Add docstring
Guidelines are CamelCases (ArrayDict)